sys/crypto/tweetnacl/tweetnacl.c

```diff
@@ -0,0 +1,810 @@
+#include "tweetnacl.h"
```

Comment:
https://github.com/RIOT-OS/RIOT/pull/4317/files#diff-0abdd814cde24db625913f773f8bf7e6R3 mentions it's public domain, but yeah a note in the C-file might be helpful, too.